lib/geometry/polygon.nit

```diff
@@ -19,9 +19,8 @@ module polygon
 
 import points_and_lines
 
-# Convex Polygon class
-class ConvexPolygon
-
+# Abstraction of a Polygon
+abstract class APolygon
```

Comment:
Well I guess I could rename `Polygon` into `SimplePolygon` since I'm not planning on handling complex polygons and then rename `APolygon` intro `Polygon`.